libs/facilities/frontend/view/src/lib/components/browse/browse.page.html

```diff
@@ -1 +1,20 @@
-<p>browse works!</p>
+<ix-content>
+	<ix-content-header slot="header" header-title="Main Dashboard" [headerSubtitle]="subtitle">
+	</ix-content-header>
+
+	<ix-card-list label="Facilities Overview" [showAllCount]="facilities.length" list-style="stack">
+		@for (facility of facilities; track facility){
```

Comment:
Usually u track the id not the whole item f.e. `facility.id` but it could also be a differnent property like pumps.